services/web-app/config/constants.js

```diff
@@ -4,14 +4,18 @@
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
-import { getRunMode, RunMode } from '@carbon-platform/api/runtime'
+import { getRunMode, loadEnvVars, RunMode } from '@carbon-platform/api/runtime'
 
-export const CACHE_PATH = getRunMode() === RunMode.Standard ? '.carbon' : 'public/data'
-export const IMAGES_CACHE_PATH = '.carbon'
+const CACHE_PATH = getRunMode() === RunMode.Standard ? '.carbon' : 'public/data'
+const IMAGES_CACHE_PATH = '.carbon'
 
-export const ALLOWED_REFERERS = [
+const ALLOWED_REFERERS = [
   { protocol: 'https:', domain: '.carbondesignsystem.com' },
   { protocol: 'https:', domain: 'web-app.j73b4w218e4.us-south.codeengine.appdomain.cloud' },
   { protocol: 'http:', domain: 'localhost' },
   { protocol: 'https:', domain: 'localhost' }
 ]
+
+const { CARBON_INTERNAL_API_SECRET } = loadEnvVars({ CARBON_INTERNAL_API_SECRET: 'DEV_SECRET' })
```

Comment:
Can you add a little blurb to the web app docs to call out this envvar? 😁

Reply:
Done!